### ISSUE TITLE
miniconda in home fails

### ISSUE THREAD

**Issue by dfroger** [heimdali/heimdali#89]
In develop mode, using miniconda in home fails.

fftw3 is search as something like: 
`/home/home/ROCQ/clime/bereziat/miniconda/envs/heimdali-dev/lib/libfftw3.so`
instead of:
`/home/ROCQ/clime/bereziat/miniconda/envs/heimdali-dev/lib/libfftw3.so`

Heimdali version: 0.7.2
OS: Ubuntu 12.04

**Comment by dfroger** [comment]
Same issue on Travis

**Comment by dfroger** [comment]
I can reproduce on my laptop.

# In the itk-heimdali conda package

In directory `/home/(...)/miniconda/pkgs/itk-heimdali-dbg-4.7.0.3-1`, files:
- `lib/cmake/ITK-4.8/ITKConfig.cmake` 
- `lib/cmake/ITK-4.8/ITKTargets-debug.cmake`

contain hardcoded paths `/home/opt/anaconda1anaconda2anaconda3/` for `fftw3`, and other libs.

File `info/has_prefix` contains:

```
/opt/anaconda1anaconda2anaconda3 text include/ITK-4.8/gdcmConfigure.h
/opt/anaconda1anaconda2anaconda3 text lib/cmake/ITK-4.8/ITKConfig.cmake
/opt/anaconda1anaconda2anaconda3 text lib/cmake/ITK-4.8/ITKTargets-debug.cmake
/opt/anaconda1anaconda2anaconda3 text lib/cmake/ITK-4.8/Modules/ITKHDF5.cmake
```

# After the itk-heimdali conda package is installed

In the directory `/home/(...)/miniconda/envs/heimdali-dev`, files
- `lib/cmake/ITK-4.8/ITKConfig.cmake` 
- `lib/cmake/ITK-4.8/ITKTargets-debug.cmake`

contain hardcoded path: `/home/home/(...)/miniconda/envs/heimdali-dev`

# Build ITK manually

When building ITK manually, exactly the same way the conda recipe does, paths in the two above files are correct (`/home/(...)/miniconda/envs/itk-build`).

**Comment by dfroger** [comment]
This is related to how `itk-heimdali` conda package has been created.

`conda-build` is installed in `/local/froger`, and `realpath /local/froger` is `/home/local/froger`

**Comment by dfroger** [comment]
Fixed in 0.7.3, conda package are now build on a Centos 5.11 Vagrant [box](https://github.com/dfroger/conda-build-env-gnulinux)
